### DISCUSSION TITLE
Nextjs Commerce Template doesn't work on Hobby Plan

### DISCUSSION THREAD

**Discussion by digicoparis** [vercel/community#3042] [Idea]
Hello,

I started Nextjs Commerce Template to give a try about performance, but every vercel deploy command says that Edge Functions are too big for the Hobby plan (by a few 50/100Kb).

Please state in
https://vercel.com/templates/next.js/nextjs-commerce
landing page that it needs a Pro Plan.

[image](https://github.com/vercel/community/assets/25030291/36a0634e-750f-44cf-a669-f71871889908)

Best regards,
@digicoparis

**Comment by manovotny** [comment]
We are tracking this issue in the Commerce repo. Please subscribe to this issue for updates: https://github.com/vercel/commerce/issues/1047

We are [working to get bundle size down](https://github.com/vercel/commerce/pull/1054) so it works on all Vercel plans.

**Reply by digicoparis** [reply]
Thank you, I'll give it another try on vercel deploy.

**Reply by manovotny** [reply]
@digicoparis to be clear, we don't have it solved yet, which is why I linked you to an issue and PR so you could follow along and subscribe to updates.

**Comment by digicoparis** [comment]
@michealnovotny ok, many thanks for updates.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Sans
virus.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Le mer. 28 juin 2023 à 23:42, Michael Novotny ***@***.***> a
écrit :

> @digicoparis <https://github.com/digicoparis> to be clear, we don't have
> it solved yet, which is why I linked you to an issue and PR so you could
> follow along and subscribe to updates.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/3042#discussioncomment-6309206>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF665EYAA7VDM4T4EYQYXZ3XNSQMDANCNFSM6AAAAAAZXSYLFY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
François-Yves Prigent
06.68.55.48.91
